Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "utils/analytics"
+require "formula_installer"
+
+describe Utils::Analytics do
+  describe "::os_prefix_ci" do
+    context "when anonymous_os_prefix_ci is not set" do
+      before(:each) do
+        if described_class.instance_variable_defined?(:@anonymous_os_prefix_ci)
+          described_class.send(:remove_instance_variable, :@anonymous_os_prefix_ci)
```

Comment:
Need to try and figure out a way to not set/get instance variables here. If need be: delete the relevant tests that need them set given we're almost at the end of GSoC. Ideally: just call other methods that end up calling these ones.